dts/bindings/display/sitronix,st7735r.yaml

```diff
@@ -117,3 +117,9 @@ properties:
       description: Display Inversion Control
 
         Set dot inversion or line inversion for each normal/idle/partial mode.
+
+    inversion-on:
+      type: boolean
+      description: Enable Display Inversion
```

Comment:
Not sure about style here, we have different in the tree, but @mbolivar-nordic will complain if necessary.
```
      description: |

        Enable Display Inversion
        Make a drawing with the inverted color of the frame memory.
```

Reply:
@jfischer-no is right; this needs to be done in a different way. Like this, please:

```
description: |
    Enable Display Inversion
    Make a drawing with the inverted color of the frame memory.
```

The same change should be made for the `invctr` property also.

Please see https://docs.zephyrproject.org/latest/build/dts/bindings.html#general-rules for an explanation about why this is needed.

Reply:
@jfischer-no , @mbolivar-nordic 
Thanks for reviewing.
I Fixed it.